custom_components/nhc2/nhccoco/devices/generic_hvac.py

```diff
@@ -39,6 +41,10 @@ def supports_program(self) -> bool:
     def operation_mode(self) -> str:
         return self.extract_property_value(PROPERTY_OPERATION_MODE)
 
+    @property
+    def possible_operation_modes(self) -> List[str]:
```

Comment:
Why this `List` and not `list`. 
I'm not a python expert, and throughout the codebase `list` is used. Can you change this for the sake of having consitency

Reply:
This one is still `List`